acceptance/tests/reports/corrective_change_new_resource.rb

```diff
@@ -0,0 +1,83 @@
+require 'yaml'
+
+test_name "C98092: corrective change new resource" do
+require 'puppet/acceptance/environment_utils'
+extend Puppet::Acceptance::EnvironmentUtils
+
+  app_type = File.basename(__FILE__, '.*')
```

Comment:
Thank you James for the valuable comments.
The comment is incorporated to the file
  test_file_name = File.basename(__FILE__, '.*')
  tmp_environment   = mk_tmp_environment_with_teardown(master, test_file_name)